hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HFileOutputFormat2.java

```diff
@@ -152,6 +153,10 @@ protected static byte[] combineTableNameSuffix(byte[] tableName, byte[] suffix)
   static final String BLOCK_SIZE_FAMILIES_CONF_KEY = "hbase.mapreduce.hfileoutputformat.blocksize";
   static final String DATABLOCK_ENCODING_FAMILIES_CONF_KEY =
     "hbase.mapreduce.hfileoutputformat.families.datablock.encoding";
+  public static final String TABLE_NAME_WITH_NAMESPACE_INCLUSIVE_KEY =
```

Comment:
can you just add a quick comment to this: 
```
// When MULTI_TABLE_HFILEOUTPUTFORMAT_CONF_KEY is enabled, should table names be written 
// with namespace included.
// This will be the behavior when upgrading to 3.0.
```